### ISSUE TITLE
Why did the preprocess_input method was removed in 1.0.0?

### ISSUE THREAD

**Issue by etudor** [qubvel/efficientnet#84]
https://github.com/qubvel/efficientnet/blob/8aa3fa5479ec006195f220287b2d18eb90405f96/efficientnet/preprocessing.py#L55

Should I use the logic and preprocess the input image myself? Or should I completely skip it?

I've tested and trained a model with the preprocess logic from the `preprocess_input` method added and it produces better initial results. 
Without it, in training, it looks stuck at < 0.01 training accuracy and it does not seem to go past it.
With it, 15% in the current epoch it's around 0.45 acc. (one downside is that it slows down 3-4 times the training)
Not that relevant as this is training but still. It looks that it's moving something.

**Comment by qubvel** [comment]
It is not disappeared, now used preprocessing from keras-applications. You can import it 
```python
from efficientnet.keras import preprocess_input
```

**Comment by sobhanhemati** [comment]
Hi
I cannot import preprocessing:
from efficientnet.keras import preprocessing
cannot import name 'preprocessing' from 'efficientnet.keras' (....anaconda3/envs/tf2-gpu/lib/python3.7/site-packages/efficientnet/keras.py)

I am using tf2.

Any idea will be appreciated

**Comment by leejaeyeop** [comment]
> Hi
> I cannot import preprocessing:
> from efficientnet.keras import preprocessing
> cannot import name 'preprocessing' from 'efficientnet.keras' (....anaconda3/envs/tf2-gpu/lib/python3.7/site-packages/efficientnet/keras.py)
> 
> I am using tf2.
> 
> Any idea will be appreciated

when using tf2, you can use tfkeras instead.
```python3
from efficientnet.tfkeras import preprocess_input
```
